docker-services.yml

```diff
@@ -35,6 +35,10 @@ services:
       - bootstrap.memory_lock=true
       - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
       - discovery.type=single-node
+      - cluster.routing.allocation.disk.threshold_enabled=true
```

Comment:
I've added this since local ES was failing all the time

Reply:
Something to be added to `invenio-app-ils` too? And in [docker-services-cli](https://github.com/inveniosoftware/docker-services-cli/blob/master/docker_services_cli/docker-services.yml#L38)?

Reply:
I'm not sure if the others have the same issues already. Maybe should be tested first by more people ?